### PR TITLE
3moArch (XenoArch Redux)

### PR DESCRIPTION
**Pull request by EmoGarbage404**
## About the PR

Based on the [hit design doc](https://docs.spacestation14.com/en/space-station-14/departments/science/proposals/xenoarch-redux.html?highlight=3moarc#xenoarch-redux-3moarch).

Opening a draft just so i can track progress and see changes over time easier. Blame git for having a decent web UI.

## Why / Balance

X

## Technical details

X

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

no changelog yet.

### CODE REVIEW

**Comment by beck-thompson** [comment]
This looks so fun! I think the artifact fragment update is my favorite part but its hard to choose because it's all super interesting. Good luck 🫡

File: Resources/Prototypes/XenoArch/effects.yml [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
+  abstract: true
+  components:
+  - type: XenoArtifactNode
+
 - type: entity
   id: XenoArtifactEffectTest
-  name: test effect
-  noSpawn: true
+  parent: BaseXenoArtifactEffect
+  description: Bombs the middle east.
```
Comment (lzk228) [inline Resources/Prototypes/XenoArch/effects.yml:13]: whaaaat:trollface:

Comment (EmoGarbage404) [reply Resources/Prototypes/XenoArch/effects.yml]: the artifacts were not very respectful of modern geopolitics. this has been remedied.

**Comment by TheShuEd** [comment]
Emo has left us. His creation now continues to be created in a new PR:
https://github.com/space-wizards/space-station-14/pull/33370

**Pull request closed without merging**